collect_app/src/main/java/org/odk/collect/android/geo/OsmDroidMapFragment.java

```diff
@@ -302,6 +302,7 @@
 
     @Override public void clearFeatures() {
         map.getOverlays().clear();
+        map.getOverlays().add(new AttributionOverlay(getContext()));
```

Comment:
It looks like we have duplicated work we need to do to set up the view in both `clearFeatures` and `onCreateView` (this line and the line below for instance). I think it'd be good to extract this to a shared method so we're less likely to run into inconsistencies in the future. Does that make sense?

Reply:
Done. 
If you have better idea for the method name, I whoud be happy to update PR

Reply:
👍 Seems good for now!